sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/CreateMessageBatchOptions.cs

```diff
@@ -12,7 +12,7 @@ namespace Azure.Messaging.ServiceBus
     ///   behaves and is sent to the Queue/Topic.
     /// </summary>
     ///
-    public class CreateBatchOptions
+    public class CreateMessageBatchOptions
```

Comment:
I'm not sure that this rename offers much.  Were we concerned that there would be confusion with another set of batch creation options?  

I wouldn't think that this type is something that developers would need to discover from a list on its own, but would instead be guided by the parameter type/name when explicitly calling to create a batch.  At that point, I think the purpose is clear.

Reply:
Well, we renamed the operation to CreateMessageBatchAsync, so this maintains consistency with that. Do you have the same concerns with renaming to CreateMessageBatchAsync?

Reply:
I do.  I'm bucketing them both under the question "does adding more words add more clarity or just give readers more things to parse?"   That said, I don't have particularly strong feelings.

Reply:
I think it adds more clarity and also consistency. We are now adding Message/Messages for all service operations that operate on a message. It would be strange to not do the same here.